packages/protocol-kit/src/utils/signatures/utils.ts

```diff
@@ -1,11 +1,15 @@
 import { recoverAddress } from 'viem'
 import SafeProvider from '@safe-global/protocol-kit/SafeProvider'
-import { SafeSignature, SafeEIP712Args, SafeTransactionData } from '@safe-global/types-kit'
-import semverSatisfies from 'semver/functions/satisfies.js'
+import {
+  SafeSignature,
+  SafeEIP712Args,
+  SafeTransactionData,
+  SigningMethod
+} from '@safe-global/types-kit'
+import semverSatisfies from 'semver/functions/satisfies'
```

Comment:
```suggestion
import semverSatisfies from 'semver/functions/satisfies.js'
```

`.js` was added to this import for dual builds to work.